### PR TITLE
Issue #113 taskCard UI

### PR DESCRIPTION
**Pull request by Michelle-932**
## Description

Fixed first half of issue #113 to re-style the task cards to reflect priority level. (Adding assignees and having instant re-rendering are not included.)

My updated readme.md and styleguide.md files are also included, so you can discard my previous unmerged PR. 

## Related Issue

-   [ ] Is this related to a specific issue? Is so please specify: Issue #113 taskCard UI

## Type of Changes

Use a check for ✓ the following type of changes:

|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
| ✓   | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

### Before

[If applicable, include a picture of the behavior before the changes were made. I.e. a screenshot of the behavior before the changes were made.]

### After

image

## Testing Steps / QA Criteria

Additional testing needs to be done for stability reasons -- this update is extremely unstable and potentially fails when any unrelated changes to the file are made. (No idea why.)

## Checklist:

-   [ ✓] This PR is up to date with the development branch, and merge conflicts have been resolved
-   [ ✓] My code follows the style guidelines of this project
-   [✓ ] I have executed npm run lint and resolved any outstanding errors. 
-   [ ✓] I have performed a self-review of my own code
-   [✓ ] I have commented my code, particularly in hard-to-understand areas
-   [ ✓] My changes generate no new warnings

### CODE REVIEW

**Review by IntelliJinceTech — CHANGES_REQUESTED** [review]
I was having issues seeing the accent on the card for some reason so I changed the code with just conditional rendering instead of using an object. Let me know if that works for you! 

Also added some comments to the markdown

File: README.md [hunk 1]
```diff
@@ -13,48 +22,42 @@ These instructions will get you a copy of the project up and running on your loc
...
-* Live:
-* Release:
-* Development:
+* GET /: Main route, serving basic app information.
+* GET /boards: Get all boards.
+* POST /boards: Create a new board.
+* GET /boards/:boardId: Get details of a specific board.
+* PUT /boards/:boardId: Update a specific board.
+* DELETE /boards/:boardId: Delete a specific board.
+* Other routes as needed for columns, tasks, and comments.
```
[the remark this answers is not in the document]
Comment (Michelle-932) [reply README.md:57]: Ok that's fine! We can delete that part. I was still getting the hang of who exactly the audience will be.

[the remark this answers is not in the document]
Comment (mjbramich) [reply README.md]: Hey Guys, 
Here is some good reading.
https://tailwindcss.com/docs/content-configuration#dynamic-class-names. 
I like the idea of mapping classNames to an object like they show. Slightly adjusted the code and it works
image
image

There is also a bug now where when we drag a task, we lose the contents of the task. Im not sure what has changed, so need to fix that as well.

**Comment by Michelle-932** [comment]
Yes, these changes work great! I was on the right track but thanks for coming up with the conditional rendering. 
I incorporated your changes in the latest commit, they should be showing up there.

**Comment by IntelliJinceTech** [comment]
Looks like I'm getting this weird bug: 

image

@Michelle-932 are you around to address that? If not, someone else on the team will have to take over this PR.

**Comment by Michelle-932** [comment]
Yes, please reassign this PR. Thank you

**Pull request closed without merging**